lib/rules/order-in-components.js

```diff
@@ -1,60 +1,72 @@
 'use strict';
 
-var utils = require('../utils/utils');
 var ember = require('../utils/ember');
+var reportUnorderedProperties = require('../utils/property-order').reportUnorderedProperties;
+
+const ORDER = [
+  'service',
+  'property',
+  'single-line-function',
+  'multi-line-function',
+  'observer',
+  'lifecycle-hook',
+  'actions',
+  'method',
+  'unknown',
+];
+
+const NAMES = {
+  'service': 'service injection',
```

Comment:
yeah, we could probably do that in a separate refactoring. I would also like to add the names of the properties to the type descriptions if possible, but I wanted to get this PR merged first.